src/browser/api/application.js

```diff
@@ -777,6 +777,25 @@ Application.setShortcuts = function(identity, config, callback, errorCallback) {
     }
 };
 
+Application.setAppLogUsername = function(identity, username, callback, errorCallback) {
```

Comment:
Since this is a new api call, let's return a promise rather than using callbacks

Reply:
if this is the old adapter idk, probably better to be consistent here with the callbacks